app/src/main/res/layout-sw600dp-port/profile_progress_header.xml

```diff
@@ -46,13 +47,13 @@
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="44dp"
-      android:layout_marginEnd="42dp"
       android:layout_marginTop="12dp"
+      android:layout_marginEnd="42dp"
       android:fontFamily="sans-serif-medium"
       android:text="@{viewModel.profile.name}"
+      android:textAlignment="viewStart"
```

Comment:
Remove this. Though it was added but for all xml files we need to remove `textAlignment`, `textDirection` and `textLocale`. Though we are not doing it right away but whenever we see this in code we should remove it.